#162163257 Change database engine from sqlite to postgresql

Nta1e opened the pull request:
**What does this PR do?**
This pull request upgrades the database from Django's default Sqlite database to Postgresql

**Description of the tasks to be completed?**
Django comes with its default light weight sqlite database but since the project we are working on will have many users, a more convinient database(postgresql) will be required.

**How should this be manually tested?**
First install requirements with `pip install -r requirements.txt`
Create a `.env` file and put your environment variables for the database credentials with the following keys
`DB_NAME` -> _Your local database name_
`DB_USER` -> _Your local postgres user_
`DB_PASSWORD` -> _Your local postgres password_
`PORT` -> _The port on which postgres is running_

**What are the relevant pivotal tracker stories**
#162163257

Nta1e committed:
Chore(Upgrade django): Upgrade Django from 1.1 to 2.1

This commit delivers an upgrade of django from 1.1 to 2.1
because Django 1.1 is depricated

[Delivers #162163252]

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit delivers a switch of databases from the default sqlite database to postgres

[Delivers #162163257]

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit delivers a switch of databases from the default sqlite database to postgres

[Delivers #162163257]

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit fixes the database credentials to be picked from environment variables

[Fixes #162163257]

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit fixes the database credentials to be picked from environment variables

[Fixes #162163257]

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit fixes the database credentials to be picked from environment variables

[Fixes #162163257]

Nta1e committed:
chore(switchdb engine): Switch database from sqlite to postgres

This commit fixes the database credentials to be picked from environment variables

[Fixes #162163257]

collin5 commented on authors/settings.py:
```diff
@@ -80,8 +82,11 @@
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv("DB_NAME"),
+        'USER': os.getenv("DB_USER"),
```
Also add default if env variable is not set. `os.getenv("ENV_VAR", <DEFAULT_VALUE>)`.

Nta1e replied:
oh alright. Working on it

collin5 reviewed (changes requested):
Good work! One small thing before we merge this.

Nta1e committed:
chore(switch db engine): Switch database from sqlite to postgres

This commit adds default database creadentials incase someone had no environment variables set

[Fixes #162163257]

Nta1e committed:
chore(switch database engine): Switch database from sqlite to postgres

This commit sets default database credentials incase one has no environment variables set

[Fixes #162163257]
